neuralmonkey/decoders/sequence_classifier.py

```diff
@@ -18,24 +19,56 @@ class SequenceClassifier(ModelPart):
     The API pretends it is an RNN decoder which always generates a sequence of
     length exactly one.
     """
+
     # pylint: disable=too-many-arguments
     def __init__(self,
                  name: str,
                  encoders: List[Any],
                  vocabulary: Vocabulary,
                  data_id: str,
-                 layers: Optional[List[int]]=None,
-                 activation: Callable[[tf.Tensor], tf.Tensor]=tf.tanh,
-                 dropout_keep_prob: float=0.5,
-                 save_checkpoint: Optional[str]=None,
-                 load_checkpoint: Optional[str]=None) -> None:
+                 layers: List[int] = None,
+                 activation: str = 'RELU',
+                 dropout_keep_prob: float = 0.5,
+                 save_checkpoint: Optional[str] = None,
+                 load_checkpoint: Optional[str] = None) -> None:
```

Comment:
toto je hrozne divne, nikde v definici PEP8 to nemuzu najit a pokud to tam je, tak je to absurdni kdyz jinde se davaji mezery okolo rovnitek a podobnych znacek. Tak snad ted ten travis projde